lib/components/Text/styles.ts

```diff
@@ -0,0 +1,44 @@
+import { css } from "@emotion/css";
+import breakpoints from "../../theme/breakpoints";
+
+export const headingLarge = css`
+  font-size: 4.4rem;
```

Comment:
I wonder if all of this can be consolidated into the `typography` file in @klimadao/lib/theme at some point?
The typography file is useful as any other element can compose from it

```js
const newElement = css`
  ${typography.h1};
  color: red;
`
```